code/ShareCare.php

```diff
@@ -35,24 +35,17 @@ class ShareCare extends DataExtension
     private static $pinterest = false;
 
     /**
-     * Message shown at top of Share tab. Set to false to disable.
-     *
-     * @var string
-     * @config
-     */
-    private static $cms_message = 'When this page is shared by people on social media it will look something like this:';
-
-    /**
      * Add a Social Media tab with a preview of share appearance to the CMS.
      */
     public function updateCMSFields(FieldList $fields)
     {
-        $msg = Config::inst()->get('ShareCare', 'cms_message');
+		$msg = _t('ShareCare.CMSMessage');
```

Comment:
1. Please move the default string 'When this page is shared by people on social media it will look something like this:' from $tab to $msg. Default for $tab is 'Share'.
2. Spaces for indentation please on PHP files. A few tabs have been introduced in ShareCareFields.php and ShareCare.php (just find/replace with 4 spaces).